scripts/consensus.py

```diff
@@ -1,9 +1,11 @@
+from distutils.command.build import build
```

Comment:
I started commenting on this script, but then I realized that you might have included it by mistake.

This looks way too specific to the Cell health data than what we are looking for in a generic tool.

Reply:
Yeah. it's Vscode auto import...